explorer/src/test/java/com/scalar/client/tool/explorer/ExplorerTest.java

```diff
@@ -175,48 +140,55 @@ public void scan_ThrowExceptionOnlyOneTime_ShouldReturnSuccess() {
   public void scan_FailedResponseWhenRegisteringCertificate_ShouldThrowExplorerException() {
     // Arrange
     when(clientService.executeContract(eq(HOLDER_ID + "SCAN"), any(JsonObject.class)))
-        .thenReturn(contractExecutionResponseCertificateNotFound);
-    when(clientService.registerCertificate()).thenReturn(ledgerServiceResponseCertificateNotFound);
+        .thenThrow(clientExceptionCertificateNotFound);
+    doAnswer((i) -> { return null;} ).when(clientService).registerCertificate();
```

Comment:
Ditto.

Reply:
Thank you for review comment.
Code modified with the above change.